src/console-services.yml

```diff
@@ -99,57 +175,125 @@
   description: 'EC2 - Virtual Servers in the Cloud'
   url: 'https://console.aws.amazon.com/ec2/home'
 
+- command: ecr
+  description: 'ECR - Fully-managed Docker container registry'
+  url: 'https://console.aws.amazon.com/ecr/home'
+
 - command: ecs
-  description: 'EC2 Container Service - Run and Manage Docker Containers'
+  description: 'ECS - Run and Manage Docker Containers'
   url: 'https://console.aws.amazon.com/ecs/home'
 
 - command: efs
-  description: 'Elastic File System - Fully Managed File System for EC2'
+  description: 'EFS - Managed File Storage for EC2'
   url: 'https://console.aws.amazon.com/efs/home'
 
+- command: eks
+  description: 'EKS - Managed Kubernetes on AWS'
+  url: 'https://console.aws.amazon.com/eks/home'
+
 - command: elasticache
   description: 'ElastiCache - In-Memory Cache'
   url: 'https://console.aws.amazon.com/elasticache/home'
 
-- command: elasticbeanstalk
-  description: 'Elastic Beanstalk - Run and Manage Web Apps'
-  url: 'https://console.aws.amazon.com/elasticbeanstalk/home'
-
-- command: elasticmapreduce
-  description: 'EMR - Managed Hadoop Framework'
-  url: 'https://console.aws.amazon.com/elasticmapreduce/home'
+- command: elasticsearchservice
+  description: 'Elasticsearch Service -  Run and Scale Elasticsearch Clusters'
+  url: 'https://console.aws.amazon.com/elasticsearchservice/home'
 
 - command: elastictranscoder
   description: 'Elastic Transcoder - Easy-to-Use Scalable Media Transcoding'
   url: 'https://console.aws.amazon.com/elastictranscoder/home'
 
-- command: es
-  description: 'Elasticsearch Service - Run and Scale Elasticsearch Clusters'
-  url: 'https://console.aws.amazon.com/es/home'
```

Comment:
seems like i'll probably need to bump the major version for these renames, since i'm not sure who all is actually using which service (although i agree that this one in particular is probably better named as expanded)

Reply:
Maybe not a bad idea as all the icons have been updated to the new design.

Reply:
What about a command "aes" instead of "es"?

Reply:
how about "ess"? it doesn't seem to conflict, and seems easier to remember with the letters matching the service without the unrelated "a"